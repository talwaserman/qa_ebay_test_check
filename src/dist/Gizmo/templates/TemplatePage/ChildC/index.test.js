"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _reactTestRenderer = _interopRequireDefault(require("react-test-renderer"));

var _context = require("../context");

var _index = _interopRequireDefault(require("./index"));

it('Gizmo shared - ChildC', function () {
  var tree = _reactTestRenderer.default.create( /*#__PURE__*/_react.default.createElement(_context.PageContextProvider, null, /*#__PURE__*/_react.default.createElement(_index.default, null))).toJSON();

  expect(tree).toMatchSnapshot();
});