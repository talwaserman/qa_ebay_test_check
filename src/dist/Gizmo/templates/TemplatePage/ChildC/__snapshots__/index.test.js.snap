// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gizmo shared - ChildC 1`] = `
<div
  className="child-c-wrapper"
>
  child c - not connected to context
</div>
`;
